apps/backoffice-v2/src/pages/Entities/hooks/useEntities/useEntities.tsx

```diff
@@ -20,15 +21,34 @@ export const useEntities = () => {
 
   const {
     meta: { totalPages },
-    data: cases,
+    data: _cases,
   } = data || { meta: { totalPages: 0 }, data: [] };
   const entity = useEntityType();
   const individualsSearchOptions = ['entity.name', 'entity.email'];
   const businessesSearchOptions = ['entity.name'];
   const { searched, onSearch, search } = useSearch({
-    data: cases,
+    data: _cases,
     searchBy: entity === 'individuals' ? individualsSearchOptions : businessesSearchOptions,
   });
+  const { data: users } = useUsersQuery();
+  const cases = useMemo(() => {
+    const usersMapById = users?.reduce((map, user) => {
+      map[user.id] = user;
+      return map;
+    }, {});
+
+    const cases = searched.map(searchedItem => {
+      //@ts-ignore
```

Comment:
Why the `//@ts-ignore`?

Reply:
I used it because types are missing for this part.But good point,it doesnt do anything so i will remove it.